tests/components/notify/test_command_line.py

```diff
@@ -40,6 +41,7 @@ def test_bad_config(self):
 
     def test_command_line_output(self):
         """Test the command line output."""
+        ha.SERVICE_CALL_LIMIT = 1
```

Comment:
`SERVICE_CALL_LIMIT` is the maximum amount of seconds that we wait for an event that contains the result of this service.

Changing this value being the fix means that `blocking=True` reaches the end of that limit instead of returning right when the service is done. This service is calling a simple method, which should be done very fast.

Also, we should use `@patch.object('homeassistant.SERVICE_CALL_LIMIT', return_value=1)` to ensure that the value is restored after this test is done running.